src/index.css

```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+  background-color: #3f379f;
```

Comment:
NIT: Refactor to all caps to match `tailwing.config.js`

Reply:
This apparently is prettier's styling and there doesn't seem to be a flag to switch to fix, since they're "opinionated" and all. Trying to find a fix but may not get one that's simple enough

Reply:
This remains unresolved, I haven't been able to find an easy way to do this.